example_usage/run.py

```diff
@@ -102,18 +103,21 @@ def my_generator(x_train, y_train, batch_size):
         y_batch, _ = mask_generator.next()
         yield x_batch, y_batch
 
+semantic_loss = Semantic_loss_functions()
+hybrid_loss = HybridLossFunctions()
 
-s = Semantic_loss_functions()
 model = get_small_unet_no_pool()
-model.compile(optimizer=Adam(lr=1e-3), loss=s.focal_tversky,
-              metrics=[s.dice_coef, s.sensitivity, s.specificity])
+model.compile(optimizer=Adam(lr=1e-3),
+              # loss=semantic_loss.basnet_hybrid_loss,
+              loss=hybrid_loss.basnet_hybrid_loss,
```

Comment:
move all new functions to loss_functions.py, call the basnet_hybrid_loss from semantic_loss/s object.

Reply:
In next commit, I am goanna move all loss functions to `loss_functions.py` file and call them with semantic_loss/s object.

But I cannot add reference of new loss functions to summary.png.  So I guess we have to replace it with table.
If you have any work around in mind, then kindly share it.

Reply:
I can see you have updated the README.md, don't delete the summar.png, it is anyways not being used in README.md after your changes.